avocado/core/test.py

```diff
@@ -416,7 +416,7 @@ def get_state(self):
             self._update_time_elapsed()
         preserve_attr = ['basedir', 'debugdir', 'depsdir', 'fail_reason',
                          'logdir', 'logfile', 'name', 'resultsdir', 'srcdir',
-                         'status', 'sysinfodir', 'text_output', 'time_elapsed',
```

Comment:
Doesn't this removal have consequences? Or it's something that was needed in the past and it's not needed anymore?